### PR TITLE
Use fallback image for broken thumbnails

### PR DESCRIPTION
**Pull request by Ruk33**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: 3290

## What is the current behavior?

When a broken or invalid image is used for a thumbnail, no default/fallback is being used.

## What is the new behavior?

When a broken or invalid image is used for a thumbnail, a default/fallback image will be used instead.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const config = {
   PINNED_LABEL_1: process.env.PINNED_LABEL_1,
   PINNED_URI_2: process.env.PINNED_URI_2,
   PINNED_LABEL_2: process.env.PINNED_LABEL_2,
+  FALLBACK_THUMBNAIL_FALLBACK: process.env.FALLBACK_THUMBNAIL_FALLBACK,
```
Comment (seanyesmunt) [inline config.js:33]: Lets just name this `THUMBNAIL_FALLBACK`

**Comment by tzarebczan** [comment]
Thanks so much for the PR! Once we get this reviewed and merged, can you show you some [appreciation](https://lbry.com/faq/appreciation)?

File: config.js (hunk 1 above)
Comment (Ruk33) [reply config.js]: Thank you for the comment @seanyesmunt , updated.

**Comment by Ruk33** [comment]
Hello @tzarebczan , thank you for the link. Will follow the instructions shortly. Regards.

**Comment by kauffj** [comment]
@seanyesmunt @Ruk33 I think we want two fallbacks, one for streams (files), and one for channels.

**Comment by Ruk33** [comment]
Hello @kauffj , yes I've left a comment about that I wasn't sure how to test the "stream" files in the app, how can I reach that section? Also, should I update the title of the PR to WIP until done or how do you guys handle work in progress PRs?

Regards.

**Comment by kauffj** [comment]
@Ruk33 you would probably have to look at what components call `<FileThumbnail>` and add a parameter to override the fallback/default image.

**Comment by seanyesmunt** [comment]
@Ruk33 I'm going to close this but feel free to re-open if you plan on finishing this. It would be great to have in the app.

Sorry I missed your message but generally we keep PR's in a "draft" state until they are ready for review.

**Pull request closed without merging**